flexmeasures/ui/utils/breadcrumb_utils.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from flexmeasures import Sensor, Asset, Account
+from flask import url_for
+
+
+def get_breadcrumb_info(entity: Sensor | Asset | Account | None) -> dict:
+    return {
+        "breadcrumb": get_ancestry(entity),
+    }
```

Comment:
This could return a `list[dict]` instead, so just `return get_ancestry(entity)`, and this function then becomes obsolete. I think one of my earlier comments may have put you on this track, sorry.

Reply:
No worries, I'm closing this conversation as resolved and let's continue it in the other comment.